sheepdog/utils/transforms/graph_to_doc.py

```diff
@@ -153,6 +153,8 @@ def list_to_comma_string(val):
     """
 
     if val is None:
+        if file_format == "tsv":
+            return ""
         return val
```

Comment:
please add a comment for this block to explain what's going on, which values we want to return for json and tsv